src/blocks/server.blocks/middleware/__html-cache/middleware__html-cache.node.js

```diff
@@ -30,6 +30,9 @@ modules.define('middleware__html-cache', ['logger', 'constants', 'config'],
                 var raw = fs.createReadStream(pagePath),
                     acceptEncoding = req.headers['accept-encoding'] || '';
 
+                res.contentType('text/html; charset=UTF-8');
+                res.removeHeader('X-Powered-By');
```

Comment:
@tadatuta I switch off it on page - https://github.com/bem/bem-site-engine/pull/138/files#diff-9df87d4252d418b443fe2b8310557ddbR103